centreon/www/front_src/src/Resources/Actions/Resource/Check/Text.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from 'tss-react/mui';
+import { useTranslation } from 'react-i18next';
+import { equals } from 'ramda';
+
+import { Typography } from '@mui/material';
+
+const useStyles = makeStyles()((theme) => ({
+  description: {
+    fontSize: '0.8rem',
+    whiteSpace: 'pre-line'
+  },
+  title: {
+    color: theme.palette.info.main,
+    fontWeight: 'bold'
+  }
+}));
+
+interface Props {
+  description: string;
+  title: string;
+}
+
+const Text = ({ title, description }: Props): JSX.Element => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+
+  const formatLabel = (label: string): string => {
+    return label.charAt(0).toUpperCase() + label.slice(1).toLowerCase();
+  };
+
+  const formatDescription = (data: string): string => {
+    const words = data.split(' ');
+
+    const wordsWithLineBreak = words.map((word, index) => {
+      return equals(index, Math.round(words.length / 2)) ? `${word}\n` : word;
+    });
+
+    return wordsWithLineBreak.join(' ');
+  };
```

Comment:
here I have the same question, why do you want to split the string in half ? 
why not just use CSS and let the text flow naturally.

Reply:
on the ticket we should split the text on half , a cc is already useed here to achieve the point

Reply:
Where this behavior is described ? 
From what I understand, you have a title and a description. That's what is described in the ticket.

Reply:
you made 3 feedback for the same thing